README.md

```diff
@@ -65,37 +72,6 @@ Prerequisite: Having a running thyra-server application on your machine.
 To access the website you need to go to http://localhost:8080/website?url=<address of the website>.
 For instance, to access flappy text stored on the blockchain, click the following link: http://localhost:8080/website?url=A1aMywGBgBywiL6WcbKR4ugxoBtdP9P3waBVi5e713uvj7F1DJL.
```

Comment:
Need to change that contract address with the right flappy address

Reply:
FYI ATM we don't have a way to deploy automatically a website on a choosen address

Reply:
you are tight, so this address would work only in the right massa network. Should we specify the network this URL is supposed to work?